src/Bicep.Core/Features/IFeatureProvider.cs

```diff
@@ -31,8 +31,6 @@ public interface IFeatureProvider
 
     bool ExtendableParamFilesEnabled { get; }
 
-    bool SecureOutputsEnabled { get; }
```

Comment:
As discussed, I would like to enable language version 2.0 if this feature is used, as a means of moving users across. To do this, you would need to add detection here: https://github.com/Azure/bicep/blob/c7a482ab674158dd19c87b811e7f8418599ecc96/src/Bicep.Core/Emit/EmitterSettings.cs#L18-L49